go.mod

```diff
@@ -31,6 +31,7 @@ require (
 	github.com/daixiang0/gci v0.9.0
 	github.com/diskfs/go-diskfs v1.2.1-0.20210727185522-a769efacd235
 	github.com/form3tech-oss/jwt-go v3.2.3+incompatible
+	github.com/ghodss/yaml v1.0.0
```

Comment:
This package doesn't seem to be mainted anymore. Can you replace it with `sigs.k8s.io/yaml`? See https://github.com/openshift/installer/pull/6935

Reply:
fixed

Reply:
Thanks! It looks good now.